Loader/SymfonyFixturesLoader.php

```diff
@@ -72,6 +78,47 @@ protected function createFixture($class)
         return $this->loadedFixtures[$class];
     }
 
+    /**
+     * Returns the array of data fixtures to execute.
+     *
+     * @param string $set
+     *
+     * @return array $fixtures
+     */
+    public function getFixtures($set = null)
+    {
+        $fixtures = parent::getFixtures();
+
+        if ($set) {
+            $mapping = $this->setsFixtureMapping;
+            $fixtures = array_filter(
+                $fixtures,
+                function ($fixture) use ($mapping, $set) {
+                    return isset($mapping[$set][get_class($fixture)]) && $mapping[$set][get_class($fixture)];
```

Comment:
You only need `isset` here. No need to check the value stored in the array, it will always be `true`.